### PR TITLE
TRUNK-5115-new-4:Replace use of conditional logging with slf4j parame…

### PR DESCRIPTION
**Pull request by gracebish**
…terized messages
**Pull request title**
TRUNK-5115-new-4:Replace use of conditional logging with slf4j parameterized messages
**JiraIssueNumber: JiraIssueTitle**
TRUNK-5115:Replace use of conditional logging with slf4j parameterized messages
**Description of what I changed**
Replace use of conditionals which are aimed at preventing to construct the debug message with slf4js parametrized messages
**Issue I worked on**
https://issues.openmrs.org/browse/TRUNK-5115
## Checklist: I completed these to help reviewers :)

- [x] My IDE is configured to follow the [**code style**](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.

  No? Unsure? -> [configure your IDE](https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE), format the code and add the changes with `git add . && git commit --amend`

- [x] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`

- [x] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.

  No? -> execute above command

- [x] All new and existing **tests passed**.

  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.

- [x] My pull request is **based on the latest changes** of the master branch.

  No? Unsure? -> execute command `git pull --rebase upstream master`

### CODE REVIEW

**Comment by gracebish** [comment]
@dkayiwa , i have made more changes. Awaiting review.

**Comment by HerbertYiga** [comment]
@gracebish  well done on this,on your tiecket description simply add the ticket Id and the description than adding the ticketId several times

File: web/src/main/java/org/openmrs/web/filter/initialization/TestInstallUtil.java [hunk 1]
```diff
@@ -234,9 +234,7 @@ protected static boolean testConnection(String urlString) {
 			return true;
 		}
 		catch (IOException e) {
-			if (log.isDebugEnabled()) {
-				log.debug("Error generated:", e);
-			}
```
Comment (dkayiwa) [inline web/src/main/java/org/openmrs/web/filter/initialization/TestInstallUtil.java:239]: This does not look correct. There was no string concatenation here.

**Comment by gracebish** [comment]
@dkayiwa Kindly review. I have made the changes recommended.

**Pull request merged**